src/routes/users.router.ts

```diff
@@ -48,6 +48,53 @@ export const getUserByUID = usersRouter.get(
   }
 );
 
+
+/**
+ * Route to delete a user from the databases via the uid
+ */
+export const deleteUserByUID = usersRouter.delete(
+  "/deleteUserById/:uid",
+  async (req: Request, res: Response) => {
+    try {
+      const uid = req.params;
+      const result = await UserService.deleteUserByUID(uid.uid);
+      res.send(`${result.message} \n payload: ${result.payload}`);
+    } catch (error) {
+      res.send(error);
+    }
+  }
+);
+
+/**
+ * Route to delete a user from the databases via the uid
+ */
+export const deleteAllUsers = usersRouter.delete(
+  "/deleteAllUsers",
+  async (req: Request, res: Response) => {
+    try {
+      const result = await UserService.deleteAllUsers();
+      res.send(`${result.message} \n payload: ${result.payload}`);
+    } catch (error) {
+      res.send(error);
+    }
+  }
+);
+
+/**
+ * Route to delete a user from the databases via the uid
+ */
+export const getAllUsers = usersRouter.get(
+  "/getAllUsers",
+  async (req: Request, res: Response) => {
+    try {
+      const result = await UserService.getAllUsers();
+      res.send(`${result.message} \n payload: ${result.payload}`);
```

Comment:
why are we returning a concatenated string? We want a JSON/map object returned. When the front-end picks this up, it will be unable to convert the rawJson into a usable object. Have you tested this?

If we run this, instead of returning a list of users, it will return something unusable such as the following:

"Success 
payload: [{name: 'Johnny',lastName: 'Nguyen' ...}]"

which is un-usable.  The payload should be the third parameter in Result